storage/store.go

```diff
@@ -95,6 +95,24 @@ var changeTypeInternalToRaft = map[roachpb.ReplicaChangeType]raftpb.ConfChangeTy
 var storeSchedulerConcurrency = envutil.EnvOrDefaultInt(
 	"COCKROACH_SCHEDULER_CONCURRENCY", 2*runtime.NumCPU())
 
+// RaftElectionTimeout returns the default raft election timeout, as
+// computed from the specified tick interval and the default number of
+// election timeout ticks.
+func RaftElectionTimeout(raftTickInterval time.Duration) time.Duration {
+	return time.Duration(defaultRaftElectionTimeoutTicks) * raftTickInterval
```

Comment:
RaftElectionTimeoutTicks can be changed; we shouldn't hard-code the default here.